pkg/collector/providers/file.go

```diff
@@ -24,6 +24,7 @@ type configFormat struct {
 	MetricConfig  interface{} `yaml:"jmx_metrics"`
 	LogsConfig    interface{} `yaml:"logs"`
 	Instances     []check.ConfigRawMap
+	DockerImages []string `yaml:"docker_images"` // To avoid issues if check uses former conf
```

Comment:
Let's change the comment to `// Only imported for deprecation warning`

Reply:
Much better indeed